subsys/net/lib/lwm2m/Kconfig

```diff
@@ -500,6 +500,11 @@ config LWM2M_LOCATION_OBJ_SUPPORT
 config LWM2M_GATEWAY_OBJ_SUPPORT
 	bool "LwM2M Gateway Object (25) [EXPERIMENTAL]"
 
+config LWM2M_EVENT_LOG_OBJ_SUPPORT
+	bool "LwM2M Event Log Object (20) [EXPERIMENTAL]"
+	help
```

Comment:
Experimental Kconfigs should have `select EXPERIMENTAL`  entry.

Reply:
Remove experimental, no needed in that case